style.css

```
@@ -0,0 +1,1275 @@
+  #ai-avatar { width: 36px; height: 36px; border-radius: 10px; background: linear-gradient(135deg, var(--warm), var(--amber)); display: flex; align-items: center; justify-content: center; font-size: 18px; flex-shrink: 0; }
+  #ai-title { font-family: 'Playfair Display', serif; font-size: 15px; font-weight: 400; }
+  #ai-subtitle { font-family: 'JetBrains Mono', monospace; font-size: 9px; color: rgba(255,255,255,0.45); letter-spacing: 1.5px; text-transform: uppercase; margin-top: 2px; }
+  #ai-close { background: rgba(255,255,255,0.08); border: 1px solid rgba(255,255,255,0.1); color: rgba(255,255,255,0.6); font-size: 16px; cursor: pointer; width: 30px; height: 30px; border-radius: 8px; display: flex; align-items: center; justify-content: center; transition: all 0.15s; }
+  #ai-close:hover { background: rgba(255,255,255,0.15); color: white; }
+
+  #ai-messages { flex: 1; overflow-y: auto; padding: 16px; display: flex; flex-direction: column; gap: 10px; scroll-behavior: smooth; }
+  #ai-messages::-webkit-scrollbar { width: 4px; }
+  #ai-messages::-webkit-scrollbar-thumb { background: var(--border); border-radius: 10px; }
+
+  .ai-msg { max-width: 88%; padding: 11px 15px; border-radius: 16px; font-size: 13.5px; line-height: 1.65; animation: msgIn 0.2s ease both; }
+  @keyframes msgIn { from{opacity:0;transform:translateY(8px)} to{opacity:1;transform:translateY(0)} }
```

Stylelint (17.3.0)

[error] Expected keyframe name "msgIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)